thrifty-schema/src/main/java/com/microsoft/thrifty/schema/parser/ThriftyParserPlugins.java

```diff
@@ -0,0 +1,72 @@
+package com.microsoft.thrifty.schema.parser;
+
+import java.util.UUID;
+
+/**
+ * Utility class to inject handlers to certain standard Thrifty operations.
+ */
+public final class ThriftyParserPlugins {
```

Comment:
Interesting, I must have missed this class the last time I looked at this PR.  Why would you need something other than `UUID::randomUUID` - for testing, it seems?

If so, and there aren't any larger plans for this class, have you considered simply adding a `UUIDProvider` param to a `ThriftParser` constructor, and making the parser itself responsible for assigning the UUIDs?  It would mean also changing the autovalue factory functions for the parser elements, to add a UUID, but would have the benefit of being less stateful.

I think you might have deliberately decided against doing this, so I'm curious about why.

Reply:
I added it in this followup. The primary case is testing, but one could conceivably also use this to supply their own no-op version if they don't need it and want the slight perf boost from not calculating a random one.

I opted to make it a plugin setup because I wanted to have a simple static location for both simplicity in the parser's access as well as making it public for anyone creating/modifying their own types to use if they create their own (to keep things consistent)